layouts/partials/header.html

```diff
@@ -69,4 +69,9 @@
                 </button>
             </div>
         </nav>
+        <svg viewBox="0 0 500 70" preserveAspectRatio="none" style="position: absolute;top: 4rem;left: 0;width: 100%;height: 3rem;">
```

Comment:
The content of the `style` attribute needs to go to one of the css files, I think `static.css`, otherwise it will not load as we explicitly forbid inline style, see:

https://github.com/apache/camel-website/blob/18028d3ac6756094d67bb383164724d9dc367b9e/static/.htaccess#L1329

You can test this by running the web server in Docker, see https://github.com/apache/camel-website/tree/master/support#running-apache-http